src/states/features/auth/authServices.js

```diff
@@ -16,6 +17,13 @@ const login = async (data) => {
 
 const login2FA = async (data) => {
   const response = await API.get(`${API_URL}/users/login/${data}`);
+  if (!response.data.user) {
+    localStorage.setItem('isSeller', 'true');
+    localStorage.removeItem('myToken');
```

Comment:
The tokens contain the role of the person loggedin why are you now saving this "isSeller" in the LS?

Reply:
for the seller we needed to first use 2fa before disclosing the token and storing it in LS. is seller helps us in the middle steps and not using the token there ensures the security.